service.py

```diff
@@ -5,6 +5,8 @@
 from werkzeug.utils import secure_filename
 from datetime import datetime, timedelta
 from flask import Flask
+from time import mktime
+import time
```

Comment:
@CNZedChou 

Why did you "import time" while time seems to be never used?

-Hui